core/designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/Tag.kt

```diff
@@ -67,25 +57,6 @@ fun NiaTopicTag(
                 text()
             }
         }
-        NiaDropdownMenu(
```

Comment:
Note: After the removal of this here, `NiaDropdownMenu` and `NiaDropdownMenuButton` don't have any usages in the app (only in the catalog).

Reply:
I don't know how the UI looked like previously, do you mind adding a before/after screenshot of this feature to the PR description? Thanks

Reply:
That is the question 😄  If we're not expecting to use them very soon, I'd remove them.

Updated the PR description with a _Before_ video.

Reply:
Removed!